### ISSUE TITLE
Buffer randomly jumping while the app is running

### ISSUE THREAD

**Issue by Almaz5200** [wojciech-kulik/xcodebuild.nvim#148]
It started happening recently, whenever I've got the app running, even without debugger attached, buffer starts randomly jumping whenever I scroll. I've made a video to demonstrate.
Note, that I do not press any buttons other than j and k in this clip, it jumps on it's own

https://www.youtube.com/watch?v=I7m2ZcLON9c

**Comment by wojciech-kulik** [comment]
Does it happen only if the app is running? Probably it’s because of logs being printed. Can you check with opened dapui so that you see logs?

**Comment by Almaz5200** [comment]
Yep, only if the app is running
Seems like you're right. I can see logs, and those jumps happen at exactly the same time as the log prints. Do you think anything can be done about it? I don't thing this is specific to my setup since I've completely redone my Neovim config from scratch a few days ago and it's still here

**Comment by wojciech-kulik** [comment]
I will check this

**Comment by wojciech-kulik** [comment]
In the meantime, could you also try a different Neovim version? If you use 10 try with 0.9.5 or vice-versa

**Comment by Almaz5200** [comment]
Ok, I'll check it out and get back to you
Disabling console_logs helped, and I only use logs via tail in a separate tmux pane anyway :)

**Comment by Almaz5200** [comment]
It doesn't happen on 0.9.5, so it looks like it has started after I've updated

**Comment by wojciech-kulik** [comment]
Most likely it's bug in Neovim v0.10. They had many issues with cursor. However, I will check it anyway, maybe I will find some workaround

**Comment by wojciech-kulik** [comment]
I investigated this issue and it turns out that the problem is with the latest nvim-dap-ui release. They autoscroll the buffer whenever lines are changed. But they call `vim.cmd` without checking if this buffer is actually active. Therefore the command is sent to Swift file instead of dapui console window:
https://github.com/rcarriga/nvim-dap-ui/blob/334cf3038c4756e6ab999cbac67c847fb654c190/lua/dapui/elements/console.lua#L38-L43

I created PR in nvim-dap-ui repository to fix that.

Temporary workaround would be to disable console logs or to redirect them somewhere else:
```
      console_logs = {
        enabled = true,
        filter_line = function(line)
          -- redirect logs wherever you like
          print(line)
          return false
        end,
      },
```
